docs/install/install-from-packages.md

```diff
@@ -325,21 +338,8 @@ make install
 This will take a few minutes.
 ☕️
 First the backend, then the frontend will be installed.
-At the start of the frontend installation part, you might see a prompt.
 
-```console
-Need to install the following packages:
-  mrs-developer
-Ok to proceed? (y)
-``` 
-
-Hit the {kbd}`Enter` key to proceed and install `mrs-developer`.
-
-When the process completes successfully, it will exit with a message similar to the following.
-
-```console
-✨  Done in 98.97s.
-```
+When the process completes successfully, it will exit with no message.
```

Comment:
Please restore the above section. See: https://github.com/collective/cookiecutter-plone-starter/blob/138b1391a9612a9b179b711a9e5f1e6b1ed2af85/%7B%7B%20cookiecutter.project_slug%20%7D%7D/frontend/Makefile.default#L58-L59

```suggestion

If mrs-developer is already installed, when the process completes successfully, it will exit with no message.
```